consensus/quorum/quorum.go

```diff
@@ -199,6 +201,18 @@ func (s *cIdentities) NthNext(pubKey *bls.PublicKeyWrapper, next int) (bool, *bl
 	return found, &s.publicKeys[idx]
 }
 
+// NthNextHmy return the Nth next pubkey of Harmony nodes, next can be negative number
+func (s *cIdentities) NthNextHmy(instance shardingconfig.Instance, pubKey *bls.PublicKeyWrapper, next int) (bool, *bls.PublicKeyWrapper) {
+	found := false
+
+	idx := s.IndexOf(pubKey.Bytes)
+	if idx != -1 {
+		found = true
+	}
+	idx = (idx + next) % instance.NumHarmonyOperatedNodesPerShard()
```

Comment:
add a check to make sure idx is within index bound?

Reply:
it doesn't matter, right?  It mod on the number of harmony node.

Reply:
if NumHarmonyOperatedNodesPerShard returns something off. We don't want crash.

Reply:
okay